figshare/Article.py

```diff
@@ -260,6 +261,22 @@ def __get_article_metadata_by_version(self, version, article_id):
                     if (get_response.status_code == 200):
                         version_data = get_response.json()
                         payload_size = calculate_payload_size(self.system_config, version_data)
+
+                        if payload_size == 0:
+                            if self.system_config['continue-on-error'] == "False":
+                                self.logs.write_log_in_file("error",
+                                                            f"Curation folder for article {article_id} version {version['version']} not found.",
+                                                            True)
+                                self.logs.write_log_in_file("info", "Aborting execution.", True)
+                                exit()
```

Comment:
Is this `exit()` needed? The `True` parameter in `write_log_in_file` is supposed to stop the script

Reply:
The True parameter is to display the log text on screen in addition to writing it to file.

Reply:
Ah you are right. There is a second parameter that when set to true, stops execution. You might consider using that instead

Reply:
Thanks. I'll use that.